import { HelpEntryResponse } from '../../adapters/helpConverter';

// Source: /api/help/canonical_and_isoforms
// Retrieved: 2021-10-15
// TODO: this API is unstable so the mock data here will want to be updated jira: https://www.ebi.ac.uk/panda/jira/browse/TRM-26569
const data: HelpEntryResponse = {
  id: 'canonical_and_isoforms',
  title:
    'What is the canonical sequence? Are all isoforms described in one entry?',
  lastModified: '2021-07-16',
  content:
    "\n**What is the canonical sequence?**\n\nTo reduce redundancy, the UniProtKB/Swiss-Prot policy is to describe all the protein products encoded by one gene in a given species in a single entry. We choose for each entry a canonical sequence based on at least one of the following criteria:\n\n1.  It is the most prevalent.\n2.  It is the most similar to orthologous sequences found in other species.\n3.  By virtue of its length or amino acid composition, it allows the clearest description of domains, isoforms, polymorphisms, post-translational modifications, etc.\n4.  In the absence of any information, we choose the longest sequence.\n\nNote that for organisms for which the genome sequence is completed, we choose the protein sequence derived from genome translation, unless it is clearly established that a different polymorphism is more frequent for a given position.\n\nDifferences to other sequences are documented in the 'Sequence' section (in the flat file FT keys: VAR\\_SEQ, CONFLICT, VARIANT, ...). Additional information can be found in the 'Alternative sequence' subsection.\n\nThe various UniProtKB distribution formats (Flat Text, XML, RDF/XML) display only the canonical sequence. The website's 'Sequences' section displays the canonical sequence, but for convenience it offers also a view of the isoforms that are described in the 'Alternative sequence' subsection.\n\n**Are all isoforms described in one UniProtKB/Swiss-Prot entry?**\n\nWhenever possible, all the protein products encoded by one gene in a given species are described in a single UniProtKB/Swiss-Prot entry, including isoforms generated by alternative splicing, alternative promoter usage, and alternative translation initiation (\\*). However, some alternative splicing isoforms derived from the same gene share only a few exons, if any at all, the same for some 'trans-splicing' events. In these cases, the divergence is obviously too important to merge all protein sequences into a single entry and the isoforms have to be described in separate 'external' entries.\n\nExample: [isoforms derived from the lola gene (Drosophila melanogaster)](http://www.uniprot.org/uniprot/P42284#sequences)\n\n(\\*) Important remark: Due to the increase of sequence data coming from large-scale sequencing projects, UniProtKB/TrEMBL may contain additional predicted sequences encoded by genes which are described in a UniProtKB/Swiss-Prot entry.\n\n**How can I retrieve isoform sequences?**\n\nAlternative sequences, described in either single or separate entries, are all available for Blast searches.\n\nIsoform sequences can be downloaded in FASTA format from our [FTP download index page](http://www.uniprot.org/downloads) (choose the file: 'Isoform sequences').\n\nQuery-derived sets of canonical sequences alone or canonical and isoform sequences can also be downloaded in FASTA format (see [How to retrieve sets of UniProtKB protein sequences?](http://www.uniprot.org/faq/38)).\n\nSee also:\n\n*   [What are UniProtKB's criteria for defining a CDS as a protein?](http://www.uniprot.org/faq/25)\n*   [How are protein sequence variety and protein diversity represented in UniProtKB?](http://www.uniprot.org/faq/21)\n*   [Sequence length](http://www.uniprot.org/manual/sequence_length)\n*   [Alternative products](http://www.uniprot.org/manual/alternative_products)\n*   [Alternative sequence](http://www.uniprot.org/manual/var_seq)\n*   [Sequence conflict](http://www.uniprot.org/manual/conflict)",
  categories: [
    'UniProtKB',
    'Sequence',
    'Text search',
    'Download',
    'Technical',
    'faq',
  ],
};
export default data;
