import { SearchResults } from '../../../../../shared/types/results';
import { CitationsAPIModel } from '../../../../../supporting-data/citations/adapters/citationsConverter';

// Source: uniprotkb/O43865/publications?facets=types%2Ccategories%2Cis_large_scale
// Retrieved: 2025-06-18
const mock: SearchResults<Partial<CitationsAPIModel>> = {
  facets: [
    {
      label: 'Source',
      name: 'types',
      allowMultipleSelection: true,
      values: [
        {
          label: 'UniProtKB reviewed (Swiss-Prot)',
          value: '2',
          count: 25,
        },
        {
          label: 'Computationally mapped',
          value: '1',
          count: 26,
        },
        {
          label: 'Community mapped',
          value: '0',
          count: 1,
        },
      ],
    },
    {
      label: 'Category',
      name: 'categories',
      allowMultipleSelection: true,
      values: [
        {
          value: 'Interaction',
          count: 27,
        },
        {
          value: 'Function',
          count: 14,
        },
        {
          value: 'Sequences',
          count: 14,
        },
        {
          value: 'PTM / Processing',
          count: 10,
        },
        {
          value: 'Disease & Variants',
          count: 7,
        },
        {
          value: 'Subcellular Location',
          count: 5,
        },
        {
          value: 'Expression',
          count: 3,
        },
        {
          value: 'Family & Domains',
          count: 3,
        },
        {
          value: 'Structure',
          count: 1,
        },
        {
          value: 'Unclassified',
          count: 1,
        },
      ],
    },
    {
      label: 'Study type',
      name: 'is_large_scale',
      allowMultipleSelection: false,
      values: [
        {
          label: 'Small scale',
          value: 'false',
          count: 31,
        },
        {
          label: 'Large scale',
          value: 'true',
          count: 20,
        },
      ],
    },
  ],
  results: [
    {
      citation: {
        id: '11904675',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Dekker J.W.',
          'Budhia S.',
          'Angel N.Z.',
          'Cooper B.J.',
          'Clark G.J.',
          'Hart D.N.',
          'Kato M.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '11904675',
          },
          {
            database: 'DOI',
            id: '10.1007/s00251-001-0402-z',
          },
        ],
        title:
          'Identification of an S-adenosylhomocysteine hydrolase-like transcript induced during dendritic cell differentiation.',
        publicationDate: '2002',
        journal: 'Immunogenetics',
        firstPage: '993',
        lastPage: '1001',
        volume: '53',
        completeAuthorList: true,
        literatureAbstract:
          'Dendritic cells (DC) are the professional antigen-presenting cells that initiate immune responses. While DC take up antigen, migrate to lymph nodes and present processed antigen to T lymphocytes, little is known of the intracellular biochemical pathways controlling these events. Using the differential display technique, employing the activated blood DC-like cell line L428, we isolated a cDNA induced during DC differentiation likely to have a regulatory function. This cDNA encoded a putative 530-amino-acid (aa) protein consisting of a unique hydrophilic domain (106 aa) and a domain (424 aa) similar to the methylation pathway enzyme S-adenosylhomocysteine hydrolase (AHCY). Therefore, this molecule was termed DC-expressed AHCY-like molecule (DCAL). DCAL mRNA was expressed moderately in fresh blood DC, but was not detectable in other peripheral blood mononuclear cells. DCAL mRNA increased markedly during activation of blood and skin DC (Langerhans cells), but was diminished in terminally differentiated tonsil DC. Cultured monocytes expressed little DCAL mRNA, but levels increased markedly when differentiated into DC by cytokines GM-CSF and IL-4. The DCAL gene [Chromosome (Chr) 1] and another previously identified DCAL-like molecule KIAA0828 (Chr 7) differed from the AHCY gene (Chr 20) in gene organization. Thus, DCAL may have a role in controlling critical events in DC differentiation and belong to a novel family of AHCY-like molecules.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '11904675',
          sourceCategories: ['Sequences', 'Expression'],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [MRNA] (ISOFORM 1)',
            'TISSUE SPECIFICITY',
            'DEVELOPMENTAL STAGE',
          ],
          referenceNumber: 0,
        },
      ],
      statistics: {
        reviewedProteinCount: 2,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 5,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '14702039',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Ota T.',
          'Suzuki Y.',
          'Nishikawa T.',
          'Otsuki T.',
          'Sugiyama T.',
          'Irie R.',
          'Wakamatsu A.',
          'Hayashi K.',
          'Sato H.',
          'Nagai K.',
          'Kimura K.',
          'Makita H.',
          'Sekine M.',
          'Obayashi M.',
          'Nishi T.',
          'Shibahara T.',
          'Tanaka T.',
          'Ishii S.',
          'Yamamoto J.',
          'Saito K.',
          'Kawai Y.',
          'Isono Y.',
          'Nakamura Y.',
          'Nagahari K.',
          'Murakami K.',
          'Yasuda T.',
          'Iwayanagi T.',
          'Wagatsuma M.',
          'Shiratori A.',
          'Sudo H.',
          'Hosoiri T.',
          'Kaku Y.',
          'Kodaira H.',
          'Kondo H.',
          'Sugawara M.',
          'Takahashi M.',
          'Kanda K.',
          'Yokoi T.',
          'Furuya T.',
          'Kikkawa E.',
          'Omura Y.',
          'Abe K.',
          'Kamihara K.',
          'Katsuta N.',
          'Sato K.',
          'Tanikawa M.',
          'Yamazaki M.',
          'Ninomiya K.',
          'Ishibashi T.',
          'Yamashita H.',
          'Murakawa K.',
          'Fujimori K.',
          'Tanai H.',
          'Kimata M.',
          'Watanabe M.',
          'Hiraoka S.',
          'Chiba Y.',
          'Ishida S.',
          'Ono Y.',
          'Takiguchi S.',
          'Watanabe S.',
          'Yosida M.',
          'Hotuta T.',
          'Kusano J.',
          'Kanehori K.',
          'Takahashi-Fujii A.',
          'Hara H.',
          'Tanase T.-O.',
          'Nomura Y.',
          'Togiya S.',
          'Komai F.',
          'Hara R.',
          'Takeuchi K.',
          'Arita M.',
          'Imose N.',
          'Musashino K.',
          'Yuuki H.',
          'Oshima A.',
          'Sasaki N.',
          'Aotsuka S.',
          'Yoshikawa Y.',
          'Matsunawa H.',
          'Ichihara T.',
          'Shiohata N.',
          'Sano S.',
          'Moriya S.',
          'Momiyama H.',
          'Satoh N.',
          'Takami S.',
          'Terashima Y.',
          'Suzuki O.',
          'Nakagawa S.',
          'Senoh A.',
          'Mizoguchi H.',
          'Goto Y.',
          'Shimizu F.',
          'Wakebe H.',
          'Hishigaki H.',
          'Watanabe T.',
          'Sugiyama A.',
          'Takemoto M.',
          'Kawakami B.',
          'Yamazaki M.',
          'Watanabe K.',
          'Kumagai A.',
          'Itakura S.',
          'Fukuzumi Y.',
          'Fujimori Y.',
          'Komiyama M.',
          'Tashiro H.',
          'Tanigami A.',
          'Fujiwara T.',
          'Ono T.',
          'Yamada K.',
          'Fujii Y.',
          'Ozaki K.',
          'Hirao M.',
          'Ohmori Y.',
          'Kawabata A.',
          'Hikiji T.',
          'Kobatake N.',
          'Inagaki H.',
          'Ikema Y.',
          'Okamoto S.',
          'Okitani R.',
          'Kawakami T.',
          'Noguchi S.',
          'Itoh T.',
          'Shigeta K.',
          'Senba T.',
          'Matsumura K.',
          'Nakajima Y.',
          'Mizuno T.',
          'Morinaga M.',
          'Sasaki M.',
          'Togashi T.',
          'Oyama M.',
          'Hata H.',
          'Watanabe M.',
          'Komatsu T.',
          'Mizushima-Sugano J.',
          'Satoh T.',
          'Shirai Y.',
          'Takahashi Y.',
          'Nakagawa K.',
          'Okumura K.',
          'Nagase T.',
          'Nomura N.',
          'Kikuchi H.',
          'Masuho Y.',
          'Yamashita R.',
          'Nakai K.',
          'Yada T.',
          'Nakamura Y.',
          'Ohara O.',
          'Isogai T.',
          'Sugano S.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '14702039',
          },
          {
            database: 'DOI',
            id: '10.1038/ng1285',
          },
        ],
        title:
          'Complete sequencing and characterization of 21,243 full-length human cDNAs.',
        publicationDate: '2004',
        journal: 'Nat. Genet.',
        firstPage: '40',
        lastPage: '45',
        volume: '36',
        completeAuthorList: true,
        literatureAbstract:
          'As a base for human transcriptome and functional genomics, we created the "full- length long Japan" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '14702039',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Kidney',
              type: 'TISSUE',
            },
            {
              value: 'Thalamus',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 2)',
          ],
          referenceNumber: 1,
        },
      ],
      statistics: {
        reviewedProteinCount: 10288,
        unreviewedProteinCount: 2613,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '17974005',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Bechtel S.',
          'Rosenfelder H.',
          'Duda A.',
          'Schmidt C.P.',
          'Ernst U.',
          'Wellenreuther R.',
          'Mehrle A.',
          'Schuster C.',
          'Bahr A.',
          'Bloecker H.',
          'Heubner D.',
          'Hoerlein A.',
          'Michel G.',
          'Wedler H.',
          'Koehrer K.',
          'Ottenwaelder B.',
          'Poustka A.',
          'Wiemann S.',
          'Schupp I.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '17974005',
          },
          {
            database: 'DOI',
            id: '10.1186/1471-2164-8-399',
          },
        ],
        title: 'The full-ORF clone resource of the German cDNA consortium.',
        publicationDate: '2007',
        journal: 'BMC Genomics',
        firstPage: '399',
        lastPage: '399',
        volume: '8',
        completeAuthorList: true,
        literatureAbstract:
          '<h4>Background</h4>With the completion of the human genome sequence the functional analysis and characterization of the encoded proteins has become the next urging challenge in the post-genome era. The lack of comprehensive ORFeome resources has thus far hampered systematic applications by protein gain-of- function analysis. Gene and ORF coverage with full-length ORF clones thus needs to be extended. In combination with a unique and versatile cloning system, these will provide the tools for genome-wide systematic functional analyses, to achieve a deeper insight into complex biological processes.<h4>Results</h4>Here we describe the generation of a full-ORF clone resource of human genes applying the Gateway cloning technology (Invitrogen). A pipeline for efficient cloning and sequencing was developed and a sample tracking database was implemented to streamline the clone production process targeting more than 2,200 different ORFs. In addition, a robust cloning strategy was established, permitting the simultaneous generation of two clone variants that contain a particular ORF with as well as without a stop codon by the implementation of only one additional working step into the cloning procedure. Up to 92 % of the targeted ORFs were successfully amplified by PCR and more than 93 % of the amplicons successfully cloned.<h4>Conclusion</h4>The German cDNA Consortium ORFeome resource currently consists of more than 3,800 sequence-verified entry clones representing ORFs, cloned with and without stop codon, for about 1,700 different gene loci. 177 splice variants were cloned representing 121 of these genes. The entry clones have been used to generate over 5,000 different expression constructs, providing the basis for functional profiling applications. As a member of the recently formed international ORFeome collaboration we substantially contribute to generating and providing a whole genome human ORFeome collection in a unique cloning system that is made freely available in the community.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '17974005',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Brain',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 2)',
          ],
          referenceNumber: 2,
        },
      ],
      statistics: {
        reviewedProteinCount: 2472,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '16710414',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Gregory S.G.',
          'Barlow K.F.',
          'McLay K.E.',
          'Kaul R.',
          'Swarbreck D.',
          'Dunham A.',
          'Scott C.E.',
          'Howe K.L.',
          'Woodfine K.',
          'Spencer C.C.A.',
          'Jones M.C.',
          'Gillson C.',
          'Searle S.',
          'Zhou Y.',
          'Kokocinski F.',
          'McDonald L.',
          'Evans R.',
          'Phillips K.',
          'Atkinson A.',
          'Cooper R.',
          'Jones C.',
          'Hall R.E.',
          'Andrews T.D.',
          'Lloyd C.',
          'Ainscough R.',
          'Almeida J.P.',
          'Ambrose K.D.',
          'Anderson F.',
          'Andrew R.W.',
          'Ashwell R.I.S.',
          'Aubin K.',
          'Babbage A.K.',
          'Bagguley C.L.',
          'Bailey J.',
          'Beasley H.',
          'Bethel G.',
          'Bird C.P.',
          'Bray-Allen S.',
          'Brown J.Y.',
          'Brown A.J.',
          'Buckley D.',
          'Burton J.',
          'Bye J.',
          'Carder C.',
          'Chapman J.C.',
          'Clark S.Y.',
          'Clarke G.',
          'Clee C.',
          'Cobley V.',
          'Collier R.E.',
          'Corby N.',
          'Coville G.J.',
          'Davies J.',
          'Deadman R.',
          'Dunn M.',
          'Earthrowl M.',
          'Ellington A.G.',
          'Errington H.',
          'Frankish A.',
          'Frankland J.',
          'French L.',
          'Garner P.',
          'Garnett J.',
          'Gay L.',
          'Ghori M.R.J.',
          'Gibson R.',
          'Gilby L.M.',
          'Gillett W.',
          'Glithero R.J.',
          'Grafham D.V.',
          'Griffiths C.',
          'Griffiths-Jones S.',
          'Grocock R.',
          'Hammond S.',
          'Harrison E.S.I.',
          'Hart E.',
          'Haugen E.',
          'Heath P.D.',
          'Holmes S.',
          'Holt K.',
          'Howden P.J.',
          'Hunt A.R.',
          'Hunt S.E.',
          'Hunter G.',
          'Isherwood J.',
          'James R.',
          'Johnson C.',
          'Johnson D.',
          'Joy A.',
          'Kay M.',
          'Kershaw J.K.',
          'Kibukawa M.',
          'Kimberley A.M.',
          'King A.',
          'Knights A.J.',
          'Lad H.',
          'Laird G.',
          'Lawlor S.',
          'Leongamornlert D.A.',
          'Lloyd D.M.',
          'Loveland J.',
          'Lovell J.',
          'Lush M.J.',
          'Lyne R.',
          'Martin S.',
          'Mashreghi-Mohammadi M.',
          'Matthews L.',
          'Matthews N.S.W.',
          'McLaren S.',
          'Milne S.',
          'Mistry S.',
          'Moore M.J.F.',
          'Nickerson T.',
          "O'Dell C.N.",
          'Oliver K.',
          'Palmeiri A.',
          'Palmer S.A.',
          'Parker A.',
          'Patel D.',
          'Pearce A.V.',
          'Peck A.I.',
          'Pelan S.',
          'Phelps K.',
          'Phillimore B.J.',
          'Plumb R.',
          'Rajan J.',
          'Raymond C.',
          'Rouse G.',
          'Saenphimmachak C.',
          'Sehra H.K.',
          'Sheridan E.',
          'Shownkeen R.',
          'Sims S.',
          'Skuce C.D.',
          'Smith M.',
          'Steward C.',
          'Subramanian S.',
          'Sycamore N.',
          'Tracey A.',
          'Tromans A.',
          'Van Helmond Z.',
          'Wall M.',
          'Wallis J.M.',
          'White S.',
          'Whitehead S.L.',
          'Wilkinson J.E.',
          'Willey D.L.',
          'Williams H.',
          'Wilming L.',
          'Wray P.W.',
          'Wu Z.',
          'Coulson A.',
          'Vaudin M.',
          'Sulston J.E.',
          'Durbin R.M.',
          'Hubbard T.',
          'Wooster R.',
          'Dunham I.',
          'Carter N.P.',
          'McVean G.',
          'Ross M.T.',
          'Harrow J.',
          'Olson M.V.',
          'Beck S.',
          'Rogers J.',
          'Bentley D.R.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '16710414',
          },
          {
            database: 'DOI',
            id: '10.1038/nature04727',
          },
        ],
        title:
          'The DNA sequence and biological annotation of human chromosome 1.',
        publicationDate: '2006',
        journal: 'Nature',
        firstPage: '315',
        lastPage: '321',
        volume: '441',
        completeAuthorList: true,
        literatureAbstract:
          'The reference sequence for each human chromosome provides the framework for understanding genome function, variation and evolution. Here we report the finished sequence and biological annotation of human chromosome 1. Chromosome 1 is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and many other diseases. Patterns of sequence variation reveal signals of recent selection in specific genes that may contribute to human fitness, and also in regions where no function is evident. Fine-scale recombination occurs in hotspots of varying intensity along the sequence, and is enriched near genes. These and other studies of human biology and disease encoded within chromosome 1 are made possible with the highly accurate annotated sequence, as part of the completed set of chromosome sequences that comprise the reference human genome.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '16710414',
          sourceCategories: ['Sequences'],
          referencePositions: ['NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA]'],
          referenceNumber: 3,
        },
      ],
      statistics: {
        reviewedProteinCount: 1874,
        unreviewedProteinCount: 4977,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: 'CI-56FT96O07V0QG',
        citationType: 'submission',
        authors: [
          'Mural R.J.',
          'Istrail S.',
          'Sutton G.',
          'Florea L.',
          'Halpern A.L.',
          'Mobarry C.M.',
          'Lippert R.',
          'Walenz B.',
          'Shatkay H.',
          'Dew I.',
          'Miller J.R.',
          'Flanigan M.J.',
          'Edwards N.J.',
          'Bolanos R.',
          'Fasulo D.',
          'Halldorsson B.V.',
          'Hannenhalli S.',
          'Turner R.',
          'Yooseph S.',
          'Lu F.',
          'Nusskern D.R.',
          'Shue B.C.',
          'Zheng X.H.',
          'Zhong F.',
          'Delcher A.L.',
          'Huson D.H.',
          'Kravitz S.A.',
          'Mouchard L.',
          'Reinert K.',
          'Remington K.A.',
          'Clark A.G.',
          'Waterman M.S.',
          'Eichler E.E.',
          'Adams M.D.',
          'Hunkapiller M.W.',
          'Myers E.W.',
          'Venter J.C.',
        ],
        publicationDate: 'JUL-2005',
        submissionDatabase: 'EMBL/GenBank/DDBJ databases',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: 'CI-56FT96O07V0QG',
          sourceCategories: ['Sequences'],
          referencePositions: ['NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA]'],
          referenceNumber: 4,
        },
      ],
      statistics: {
        reviewedProteinCount: 537,
        unreviewedProteinCount: 2465,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '15489334',
        citationType: 'UniProt indexed literatures',
        authoringGroup: ['The MGC Project Team'],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '15489334',
          },
          {
            database: 'DOI',
            id: '10.1101/gr.2596504',
          },
        ],
        title:
          'The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).',
        publicationDate: '2004',
        journal: 'Genome Res.',
        firstPage: '2121',
        lastPage: '2127',
        volume: '14',
        completeAuthorList: true,
        literatureAbstract:
          "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.",
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '15489334',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Brain',
              type: 'TISSUE',
            },
            {
              value: 'Colon',
              type: 'TISSUE',
            },
            {
              value: 'Eye',
              type: 'TISSUE',
            },
            {
              value: 'Placenta',
              type: 'TISSUE',
            },
            {
              value: 'PNS',
              type: 'TISSUE',
            },
            {
              value: 'Testis',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] (ISOFORM 1)',
            'NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA] OF 48-251 (ISOFORM 2)',
          ],
          referenceNumber: 5,
        },
      ],
      statistics: {
        reviewedProteinCount: 32629,
        unreviewedProteinCount: 17745,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: 'CI-BIPJ46MU9NM9P',
        citationType: 'submission',
        authors: [
          'Volpe J.P.G.',
          'McDowell M.',
          'Jostes R.F.',
          'Afzal V.',
          'Sadinski W.',
          'Trask B.J.',
          'Legerski R.',
          'Cleaver J.E.',
        ],
        title:
          'Complementation of chromosomal instability in the xeroderma pigmentosum variant by a gene on human chromosome 1 with homology to S-adenosyl homocysteine hydrolase.',
        publicationDate: 'DEC-1996',
        submissionDatabase: 'EMBL/GenBank/DDBJ databases',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: 'CI-BIPJ46MU9NM9P',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Skin',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [MRNA] OF 16-530 (ISOFORM 1)',
          ],
          referenceNumber: 6,
        },
      ],
      statistics: {
        reviewedProteinCount: 1,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '12878157',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Imabayashi H.',
          'Mori T.',
          'Gojo S.',
          'Kiyono T.',
          'Sugiyama T.',
          'Irie R.',
          'Isogai T.',
          'Hata J.',
          'Toyama Y.',
          'Umezawa A.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '12878157',
          },
          {
            database: 'DOI',
            id: '10.1016/s0014-4827(03)00130-7',
          },
        ],
        title:
          'Redifferentiation of dedifferentiated chondrocytes and chondrogenesis of human bone marrow stromal cells via chondrosphere formation with expression profiling by large-scale cDNA analysis.',
        publicationDate: '2003',
        journal: 'Exp. Cell Res.',
        firstPage: '35',
        lastPage: '50',
        volume: '288',
        completeAuthorList: true,
        literatureAbstract:
          'Characterization of dedifferentiated chondrocytes (DECs) and mesenchymal stem cells capable of differentiating into chondrocytes is of biological and clinical interest. We isolated DECs and bone marrow stromal cells (BMSCs), H4-1 and H3-4, and demonstrated that the cells started to produce extracellular matrices, such as type II collagen and aggrecan, at an early stage of chondrosphere formation. Furthermore, cDNA sequencing of cDNA libraries constricted by the oligocapping method was performed to analyze difference in mRNA expression profiling between DECs and marrow stromal cells. Upon redifferentiation of DECs, cartilage-related extracellular matrix genes, such as those encoding leucine-rich small proteoglycans, cartilage oligomeric matrix protein, and chitinase 3-like 1 (cartilage glycoprotein-39), were highly expressed. Growth factors such as FGF7 and CTGF were detected at a high frequency in the growth stage of monolayer stromal cultures. By combining the expression profile and flow cytometry, we demonstrated that isolated stromal cells, defined by CD34(-), c-kit(-), and CD140alpha(- or low), have chondrogenic potential. The newly established human mesenchymal cells with expression profiling provide a powerful model for a study of chondrogenic differentiation and further understanding of cartilage regeneration in the means of redifferentiated DECs and BMSCs.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '12878157',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Chondrocyte',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [MRNA] OF 48-164 (ISOFORM 2)',
          ],
          referenceNumber: 7,
        },
      ],
      statistics: {
        reviewedProteinCount: 1,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '7782076',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Pawlak A.',
          'Toussaint C.',
          'Levy I.',
          'Bulle F.',
          'Poyard M.',
          'Barouki R.',
          'Guellaen G.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '7782076',
          },
          {
            database: 'DOI',
            id: '10.1016/0888-7543(95)80096-5',
          },
        ],
        title:
          'Characterization of a large population of mRNAs from human testis.',
        publicationDate: '1995',
        journal: 'Genomics',
        firstPage: '151',
        lastPage: '158',
        volume: '26',
        completeAuthorList: true,
        literatureAbstract:
          'We present the results of single-pass sequencing of 779 expressed sequence tags from normal human testis cDNA clones. Of the sequences generated, 319 (41%) appeared to be completely unknown and are likely to represent new genes, and 289 (37%) were identified based on exact or nonexact matches to sequences in public databases. In analyses of hybridization of four tissues, testis, brain, liver, and kidney, 6 of 12 cDNAs clones revealed testis-specific expression. This argues for the value of the combination of random sequencing and analysis of cellular expression for large-scale characterization of gene expression in the testis.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '7782076',
          sourceCategories: ['Sequences'],
          referenceComments: [
            {
              value: 'Testis',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'NUCLEOTIDE SEQUENCE [MRNA] OF 48-105 (ISOFORM 2)',
          ],
          referenceNumber: 8,
        },
      ],
      statistics: {
        reviewedProteinCount: 1,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '16754674',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Cooper B.J.',
          'Key B.',
          'Carter A.',
          'Angel N.Z.',
          'Hart D.N.J.',
          'Kato M.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '16754674',
          },
          {
            database: 'DOI',
            id: '10.1074/jbc.m602520200',
          },
        ],
        title:
          'Suppression and overexpression of adenosylhomocysteine hydrolase-like protein 1 (AHCYL1) influences zebrafish embryo development: a possible role for AHCYL1 in inositol phospholipid signaling.',
        publicationDate: '2006',
        journal: 'J. Biol. Chem.',
        firstPage: '22471',
        lastPage: '22484',
        volume: '281',
        completeAuthorList: true,
        literatureAbstract:
          'Adenosylhomocysteine hydrolase-like protein 1 (AHCYL1) is a novel intracellular protein with approximately 50% protein identity to adenosylhomocysteine hydrolase (AHCY), an important enzyme for metabolizing S-adenosyl-l- homocysteine, the by-product of S-adenosyl-l-homomethionine-dependent methylation. AHCYL1 binds to the inositol 1,4,5-trisphosphate receptor, suggesting that AHCYL1 is involved in intracellular calcium release. We identified two zebrafish AHCYL1 orthologs (zAHCYL1A and -B) by bioinformatics and reverse transcription-PCR. Unlike the ubiquitously present AHCY genes, AHCYL1 genes were only detected in segmented animals, and AHCYL1 proteins were highly conserved among species. Phylogenic analysis suggested that the AHCYL1 gene diverged early from AHCY and evolved independently. Quantitative reverse transcription-PCR showed that zAHCYL1A and -B mRNA expression was regulated differently from the other AHCY-like protein zAHCYL2 and zAHCY during zebrafish embryogenesis. Injection of morpholino antisense oligonucleotides against zAHCYL1A and -B into zebrafish embryos inhibited zAHCYL1A and -B mRNA translation specifically and induced ventralized morphologies. Conversely, human and zebrafish AHCYL1A mRNA injection into zebrafish embryos induced dorsalized morphologies that were similar to those obtained by depleting intracellular calcium with thapsigargin. Human AHCY mRNA injection showed little effect on the embryos. These data suggest that AHCYL1 has a different function from AHCY and plays an important role in embryogenesis by modulating inositol 1,4,5- trisphosphate receptor function for the intracellular calcium release.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '16754674',
          referencePositions: ['IDENTIFICATION (ISOFORM 2)'],
          referenceNumber: 9,
        },
      ],
      statistics: {
        reviewedProteinCount: 1,
        unreviewedProteinCount: 5,
        computationallyMappedProteinCount: 4,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '16793548',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Ando H.',
          'Mizutani A.',
          'Kiefer H.',
          'Tsuzurugi D.',
          'Michikawa T.',
          'Mikoshiba K.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '16793548',
          },
          {
            database: 'DOI',
            id: '10.1016/j.molcel.2006.05.017',
          },
        ],
        title:
          'IRBIT suppresses IP3 receptor activity by competing with IP3 for the common binding site on the IP3 receptor.',
        publicationDate: '2006',
        journal: 'Mol.',
        firstPage: '795',
        lastPage: '806',
        volume: 'Cell 22',
        completeAuthorList: true,
        literatureAbstract:
          'The inositol 1,4,5-trisphosphate (IP3) receptors (IP3Rs) are IP3-gated intracellular Ca2+ channels. We previously identified an IP3R binding protein, IRBIT, which binds to the IP3 binding domain of IP3R and is dissociated from IP3R in the presence of IP3. In the present study, we showed that IRBIT suppresses the activation of IP3R by competing with IP3 by [3H]IP3 binding assays, in vitro Ca2+ release assays, and Ca2+ imaging of intact cells. Multiserine phosphorylation of IRBIT was essential for the binding, and 10 of the 12 key amino acids in IP3R for IP3 recognition participated in binding to IRBIT. We propose a unique mode of IP3R regulation in which IP3 sensitivity is regulated by IRBIT acting as an endogenous "pseudoligand" whose inhibitory activity can be modulated by its phosphorylation status.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '16793548',
          sourceCategories: [
            'Function',
            'PTM / Processing',
            'Disease & Variants',
            'Interaction',
          ],
          referencePositions: [
            'FUNCTION',
            'SUBUNIT',
            'INTERACTION WITH ITPR1',
            'PHOSPHORYLATION AT SER-68 AND SER-71',
            'MUTAGENESIS OF SER-64; SER-66; SER-68; SER-70; SER-71; THR-72; SER-74; SER-76; SER-77; SER-80; THR-82; SER-84; SER-85; SER-90 AND THR-97',
          ],
          referenceNumber: 10,
        },
      ],
      statistics: {
        reviewedProteinCount: 2,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 11,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '16769890',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Shirakabe K.',
          'Priori G.',
          'Yamada H.',
          'Ando H.',
          'Horita S.',
          'Fujita T.',
          'Fujimoto I.',
          'Mizutani A.',
          'Seki G.',
          'Mikoshiba K.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '16769890',
          },
          {
            database: 'DOI',
            id: '10.1073/pnas.0602250103',
          },
        ],
        title:
          'IRBIT, an inositol 1,4,5-trisphosphate receptor-binding protein, specifically binds to and activates pancreas-type Na+/HCO3-cotransporter 1 (pNBC1).',
        publicationDate: '2006',
        journal: 'Proc. Natl. Acad. Sci. U.S.A.',
        firstPage: '9542',
        lastPage: '9547',
        volume: '103',
        completeAuthorList: true,
        literatureAbstract:
          'Inositol 1,4,5-trisphosphate (IP(3)) receptors (IP(3)Rs) are IP(3)-gated Ca(2+) channels that are located on intracellular Ca(2+) stores. We previously identified an IP(3)R binding protein, termed IP(3)R binding protein released with IP(3) (IRBIT). Because IRBIT is released from IP(3)R by physiological concentrations of IP(3), we hypothesized that IRBIT is a signaling molecule that is released from IP(3)R and regulates downstream target molecules in response to the production of IP(3). Therefore, in this study, we attempted to identify the target molecules of IRBIT, and we succeeded in identifying Na(+)/HCO(3)(-) cotransporter 1 (NBC1) as an IRBIT binding protein. Of the two major splicing variants of NBC1, pancreas-type NBC1 (pNBC1) and kidney-type NBC1 (kNBC1), IRBIT was found to bind specifically to pNBC1 and not to bind to kNBC1. IRBIT binds to the N-terminal pNBC1-specific domain, and its binding depends on the phosphorylation of multiple serine residues of IRBIT. Also, an electrophysiological analysis in Xenopus oocytes revealed that pNBC1 requires coexpression of IRBIT to manifest substantial activity comparable with that of kNBC1, which displays substantial activity independently of IRBIT. These results strongly suggest that pNBC1 is the target molecule of IRBIT and that IRBIT has an important role in pH regulation through pNBC1. Also, our findings raise the possibility that the regulation through IRBIT enables NBC1 variants to have different physiological roles.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '16769890',
          sourceCategories: ['Function', 'Disease & Variants', 'Interaction'],
          referencePositions: [
            'FUNCTION',
            'INTERACTION WITH SLC4A4',
            'MUTAGENESIS OF THR-52; THR-58; SER-62; SER-64; SER-66; SER-68; SER-70; SER-71; THR-72; SER-74; SER-76; SER-77; SER-80; THR-82; SER-84; SER-85; SER-90 AND THR-97',
          ],
          referenceNumber: 11,
        },
      ],
      statistics: {
        reviewedProteinCount: 4,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 17,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '18829453',
        citationType: 'UniProt indexed literatures',
        authors: ['He P.', 'Zhang H.', 'Yun C.C.'],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '18829453',
          },
          {
            database: 'DOI',
            id: '10.1074/jbc.m805534200',
          },
        ],
        title:
          'IRBIT, inositol 1,4,5-triphosphate (IP3) receptor-binding protein released with IP3, binds Na+/H+ exchanger NHE3 and activates NHE3 activity in response to calcium.',
        publicationDate: '2008',
        journal: 'J. Biol. Chem.',
        firstPage: '33544',
        lastPage: '33553',
        volume: '283',
        completeAuthorList: true,
        literatureAbstract:
          'Calcium (Ca2+) is a highly versatile second messenger that regulates various cellular processes. Previous studies showed that elevation of intracellular Ca2+ regulates the activity of Na+/H+ exchanger 3 (NHE3). However, the effect of Ca2+-dependent signaling on NHE3 activity varies depending on cell types. In this study, we report the identification of IP3 receptor-binding protein released with IP3 (IRBIT) as a NHE3 interacting protein and its role in regulation of NHE3 activity. IRBIT bound to the carboxyl-terminal domain of NHE3, which is necessary for acute regulation of NHE3. Ectopic expression of IRBIT resulted in Ca2+-dependent activation of NHE3 activity, whereas silencing of endogenous IRBIT resulted in inhibition of NHE3 activity. Ca2+-dependent stimulation of NHE3 activity was dependent on the binding of IRBIT to NHE3. Previously Ca2+-dependent inhibition of NHE3 was demonstrated in the presence of NHERF2. Co-expression of IRBIT was able to reverse the NHERF2-dependent inhibition of NHE3. We also showed that IRBIT-dependent activation of NHE3 involves exocytic trafficking of NHE3 to the plasma membrane and this activation was blocked by inhibition of calmodulin (CaM) or CaM-dependent kinase II. These results suggest that the overall effect of Ca2+ on NHE3 activity is balanced by IRBIT-dependent activation and NHERF2-dependent inhibition.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '18829453',
          sourceCategories: ['Function', 'Interaction'],
          referencePositions: ['FUNCTION', 'INTERACTION WITH SLC9A3'],
          referenceNumber: 12,
        },
      ],
      statistics: {
        reviewedProteinCount: 4,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '18669648',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Dephoure N.',
          'Zhou C.',
          'Villen J.',
          'Beausoleil S.A.',
          'Bakalarski C.E.',
          'Elledge S.J.',
          'Gygi S.P.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '18669648',
          },
          {
            database: 'DOI',
            id: '10.1073/pnas.0805139105',
          },
        ],
        title: 'A quantitative atlas of mitotic phosphorylation.',
        publicationDate: '2008',
        journal: 'Proc. Natl. Acad. Sci. U.S.A.',
        firstPage: '10762',
        lastPage: '10767',
        volume: '105',
        completeAuthorList: true,
        literatureAbstract:
          'The eukaryotic cell division cycle is characterized by a sequence of orderly and highly regulated events resulting in the duplication and separation of all cellular material into two newly formed daughter cells. Protein phosphorylation by cyclin-dependent kinases (CDKs) drives this cycle. To gain further insight into how phosphorylation regulates the cell cycle, we sought to identify proteins whose phosphorylation is cell cycle regulated. Using stable isotope labeling along with a two-step strategy for phosphopeptide enrichment and high mass accuracy mass spectrometry, we examined protein phosphorylation in a human cell line arrested in the G(1) and mitotic phases of the cell cycle. We report the identification of >14,000 different phosphorylation events, more than half of which, to our knowledge, have not been described in the literature, along with relative quantitative data for the majority of these sites. We observed >1,000 proteins with increased phosphorylation in mitosis including many known cell cycle regulators. The majority of sites on regulated phosphopeptides lie in [S/T]P motifs, the minimum required sequence for CDKs, suggesting that many of the proteins may be CDK substrates. Analysis of non-proline site-containing phosphopeptides identified two unique motifs that suggest there are at least two undiscovered mitotic kinases.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '18669648',
          sourceCategories: ['PTM / Processing', 'Sequences'],
          referenceComments: [
            {
              value: 'Cervix carcinoma',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-391',
            'IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS]',
          ],
          referenceNumber: 13,
        },
      ],
      statistics: {
        reviewedProteinCount: 3421,
        unreviewedProteinCount: 5044,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '19224921',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Kiefer H.',
          'Mizutani A.',
          'Iemura S.',
          'Natsume T.',
          'Ando H.',
          'Kuroda Y.',
          'Mikoshiba K.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '19224921',
          },
          {
            database: 'DOI',
            id: '10.1074/jbc.m807136200',
          },
        ],
        title:
          "Inositol 1,4,5-triphosphate receptor-binding protein released with inositol 1,4,5-triphosphate (IRBIT) associates with components of the mRNA 3' processing machinery in a phosphorylation-dependent manner and inhibits polyadenylation.",
        publicationDate: '2009',
        journal: 'J. Biol. Chem.',
        firstPage: '10694',
        lastPage: '10705',
        volume: '284',
        completeAuthorList: true,
        literatureAbstract:
          "IRBIT is a recently identified protein that modulates the activities of both inositol 1,4,5-triphosphate receptor and pancreas-type Na(+)/HCO(3)(-) cotransporter 1, and the multisite phosphorylation of IRBIT is required for achieving this modulatory action. Here, we report the identification of the cleavage and polyadenylation specificity factor (CPSF), which is a multi-protein complex involved in 3' processing of mRNA precursors, as an additional binding partner for IRBIT. We found that IRBIT interacted with CPSF and was recruited to an exogenous polyadenylation signal-containing RNA. The main target for IRBIT in CPSF was Fip1 subunit, and the phosphorylation of the serine-rich region of IRBIT was required both for direct association with Fip1 in vitro and for redistribution of Fip1 into the cytoplasm of intact cells. Furthermore, tert- butylhydroquinone (tBHQ), an agent that induces oxidative stress, increased the phosphorylation level of IRBIT in vivo and in parallel enhanced the interaction between IRBIT and CPSF and promoted the cytoplasmic distribution of endogenous Fip1. In addition to CPSF, IRBIT interacted in vitro with poly(A) polymerase (PAP), which is the enzyme recruited by CPSF to elongate the poly(A) tail, and inhibited PAP activity in a phosphorylation-dependent manner. These findings raise the possibility that IRBIT modulates the polyadenylation state of specific mRNAs, both by controlling the cytoplasmic/nuclear partitioning of Fip1 and by inhibiting PAP activity, in response to a stimulus that alters its phosphorylation state.",
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '19224921',
          sourceCategories: ['Function', 'Disease & Variants', 'Interaction'],
          referencePositions: [
            'FUNCTION',
            'INTERACTION WITH FIP1L1 AND PAPOLA',
            'RNA-BINDING',
            'MUTAGENESIS OF SER-62; SER-64; SER-66; SER-68; SER-70; SER-71; THR-72; SER-74; SER-76; SER-77; SER-80; THR-82; SER-84; SER-85; SER-90 AND THR-97',
          ],
          referenceNumber: 14,
        },
      ],
      statistics: {
        reviewedProteinCount: 6,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 16,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '19220705',
        citationType: 'UniProt indexed literatures',
        authors: ['Ando H.', 'Mizutani A.', 'Mikoshiba K.'],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '19220705',
          },
          {
            database: 'DOI',
            id: '10.1111/j.1471-4159.2009.05979.x',
          },
        ],
        title:
          'An IRBIT homologue lacks binding activity to inositol 1,4,5-trisphosphate receptor due to the unique N-terminal appendage.',
        publicationDate: '2009',
        journal: 'J. Neurochem.',
        firstPage: '539',
        lastPage: '550',
        volume: '109',
        completeAuthorList: true,
        literatureAbstract:
          'IRBIT is an inositol 1,4,5-trisphosphate (IP(3)) receptor (IP(3)R)-binding protein that inhibits the activation of IP(3)R by competing with IP(3) for the common binding site on IP(3)R. In this study, we characterize an IRBIT homologue, termed Long-IRBIT. Long-IRBIT is highly homologous to IRBIT ( approximately 88%) and heteromerizes with IRBIT. In spite of complete conservation of critical amino acids required for the interaction with IP(3)R and comparable phosphorylations on critical four Ser residues for IP(3)R- binding, Long-IRBIT retains little ability to interact with IP(3)R. Deletion mutagenesis analysis revealed that this low affinity to IP(3)R is attributable to an inhibitory effect of the Long-IRBIT specific N-terminal appendage (LISN domain). Immunohistochemical analysis revealed the distinct distribution of Long-IRBIT and IRBIT in mouse cerebellar cortex, that is, Long-IRBIT is mainly expressed in interneurons such as basket cells, while IRBIT is mainly expressed in glial cells. Furthermore, Long-IRBIT, but not IRBIT, underwent phosphorylation during neuronal differentiation in neuroblastoma cells and this phosphorylation was dependent on the LISN domain. These results suggest that Long-IRBIT has a different function from IRBIT.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '19220705',
          sourceCategories: ['PTM / Processing', 'Interaction'],
          referencePositions: [
            'INTERACTION WITH AHCYL2 AND ITPR1',
            'PHOSPHORYLATION AT SER-68; SER-71; SER-74 AND SER-77',
          ],
          referenceNumber: 15,
        },
      ],
      statistics: {
        reviewedProteinCount: 5,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 26,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '20584908',
        citationType: 'UniProt indexed literatures',
        authors: ['He P.', 'Klein J.', 'Yun C.C.'],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '20584908',
          },
          {
            database: 'DOI',
            id: '10.1074/jbc.m110.133066',
          },
        ],
        title:
          'Activation of Na+/H+ exchanger NHE3 by angiotensin II is mediated by inositol 1,4,5-triphosphate (IP3) receptor-binding protein released with IP3 (IRBIT) and Ca2+/calmodulin-dependent protein kinase II.',
        publicationDate: '2010',
        journal: 'J. Biol. Chem.',
        firstPage: '27869',
        lastPage: '27878',
        volume: '285',
        completeAuthorList: true,
        literatureAbstract:
          'Angiotensin II (ANG II) stimulates renal tubular reabsorption of NaCl by targeting Na(+)/H(+) exchanger NHE3. We have shown previously that inositol 1,4,5-triphosphate receptor-binding protein released with inositol 1,4,5- triphosphate (IRBIT) plays a critical role in stimulation of NHE3 in response to elevated intracellular Ca(2+) concentration ([Ca(2+)](i)). In this study, we investigated the role of IRBIT in mediating NHE3 activation by ANG II. IRBIT is abundantly expressed in the proximal tubules where NHE3 is located. ANG II at physiological concentrations stimulates NHE3 transport activity in a model proximal tubule cell line. ANG II-induced activation of NHE3 was abrogated by knockdown of IRBIT, whereas overexpression of IRBIT enhanced the effect of ANG II on NHE3. ANG II transiently increased binding of IRBIT to NHE3 at 5 min but became dissociated by 45 min. In comparison, it took at least 15 min of ANG II treatment for an increase in NHE3 activity and NHE3 surface expression. The stimulation of NHE3 by ANG II was dependent on changes in [Ca(2+)](i) and Ca(2+)/calmodulin-dependent protein kinases II. Inhibition of CaMKII completely blocked the ANG II-induced binding of IRBIT to NHE3 and the increase in NHE3 surface abundance. Several serine residues of IRBIT are thought to be important for IRBIT binding. Mutations of Ser-68, Ser-71, and Ser-74 of IRBIT decreased binding of IRBIT to NHE3 and its effect on NHE3 activity. In conclusion, our current findings demonstrate that IRBIT is critically involved in mediating activation of NHE3 by ANG II via a Ca(2+)/calmodulin-dependent protein kinases II-dependent pathway.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '20584908',
          sourceCategories: ['Function', 'Disease & Variants', 'Interaction'],
          referencePositions: [
            'FUNCTION',
            'INTERACTION WITH SLC9A3',
            'MUTAGENESIS OF SER-68; SER-71 AND SER-74',
          ],
          referenceNumber: 16,
        },
      ],
      statistics: {
        reviewedProteinCount: 5,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 14,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '21269460',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Burkard T.R.',
          'Planyavsky M.',
          'Kaupe I.',
          'Breitwieser F.P.',
          'Buerckstuemmer T.',
          'Bennett K.L.',
          'Superti-Furga G.',
          'Colinge J.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '21269460',
          },
          {
            database: 'DOI',
            id: '10.1186/1752-0509-5-17',
          },
        ],
        title: 'Initial characterization of the human central proteome.',
        publicationDate: '2011',
        journal: 'BMC Syst. Biol.',
        firstPage: '17',
        lastPage: '17',
        volume: '5',
        completeAuthorList: true,
        literatureAbstract:
          '<h4>Background</h4>On the basis of large proteomics datasets measured from seven human cell lines we consider their intersection as an approximation of the human central proteome, which is the set of proteins ubiquitously expressed in all human cells. Composition and properties of the central proteome are investigated through bioinformatics analyses.<h4>Results</h4>We experimentally identify a central proteome comprising 1,124 proteins that are ubiquitously and abundantly expressed in human cells using state of the art mass spectrometry and protein identification bioinformatics. The main represented functions are proteostasis, primary metabolism and proliferation. We further characterize the central proteome considering gene structures, conservation, interaction networks, pathways, drug targets, and coordination of biological processes. Among other new findings, we show that the central proteome is encoded by exon-rich genes, indicating an increased regulatory flexibility through alternative splicing to adapt to multiple environments, and that the protein interaction network linking the central proteome is very efficient for synchronizing translation with other biological processes. Surprisingly, at least 10% of the central proteome has no or very limited functional annotation.<h4>Conclusions</h4>Our data and analysis provide a new and deeper description of the human central proteome compared to previous results thereby extending and complementing our knowledge of commonly expressed human proteins. All the data are made publicly available to help other researchers who, for instance, need to compare or link focused datasets to a common background.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '21269460',
          sourceCategories: ['Sequences'],
          referencePositions: [
            'IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS]',
          ],
          referenceNumber: 17,
        },
      ],
      statistics: {
        reviewedProteinCount: 4960,
        unreviewedProteinCount: 12602,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '21406692',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Rigbolt K.T.',
          'Prokhorova T.A.',
          'Akimov V.',
          'Henningsen J.',
          'Johansen P.T.',
          'Kratchmarova I.',
          'Kassem M.',
          'Mann M.',
          'Olsen J.V.',
          'Blagoev B.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '21406692',
          },
          {
            database: 'DOI',
            id: '10.1126/scisignal.2001570',
          },
        ],
        title:
          'System-wide temporal characterization of the proteome and phosphoproteome of human embryonic stem cell differentiation.',
        publicationDate: '2011',
        journal: 'Sci. Signal.',
        firstPage: 'RS3',
        lastPage: 'RS3',
        volume: '4',
        completeAuthorList: true,
        literatureAbstract:
          'To elucidate cellular events underlying the pluripotency of human embryonic stem cells (hESCs), we performed parallel quantitative proteomic and phosphoproteomic analyses of hESCs during differentiation initiated by a diacylglycerol analog or transfer to media that had not been conditioned by feeder cells. We profiled 6521 proteins and 23,522 phosphorylation sites, of which almost 50% displayed dynamic changes in phosphorylation status during 24 hours of differentiation. These data are a resource for studies of the events associated with the maintenance of hESC pluripotency and those accompanying their differentiation. From these data, we identified a core hESC phosphoproteome of sites with similar robust changes in response to the two distinct treatments. These sites exhibited distinct dynamic phosphorylation patterns, which were linked to known or predicted kinases on the basis of the matching sequence motif. In addition to identifying previously unknown phosphorylation sites on factors associated with differentiation, such as kinases and transcription factors, we observed dynamic phosphorylation of DNA methyltransferases (DNMTs). We found a specific interaction of DNMTs during early differentiation with the PAF1 (polymerase- associated factor 1) transcriptional elongation complex, which binds to promoters of the pluripotency and known DNMT target genes encoding OCT4 and NANOG, thereby providing a possible molecular link for the silencing of these genes during differentiation.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '21406692',
          sourceCategories: ['PTM / Processing', 'Sequences'],
          referencePositions: [
            'ACETYLATION [LARGE SCALE ANALYSIS] AT SER-2',
            'PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-2',
            'CLEAVAGE OF INITIATOR METHIONINE [LARGE SCALE ANALYSIS]',
            'IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS]',
          ],
          referenceNumber: 18,
        },
      ],
      statistics: {
        reviewedProteinCount: 2149,
        unreviewedProteinCount: 3236,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '22814378',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Van Damme P.',
          'Lasa M.',
          'Polevoda B.',
          'Gazquez C.',
          'Elosegui-Artola A.',
          'Kim D.S.',
          'De Juan-Pardo E.',
          'Demeyer K.',
          'Hole K.',
          'Larrea E.',
          'Timmerman E.',
          'Prieto J.',
          'Arnesen T.',
          'Sherman F.',
          'Gevaert K.',
          'Aldabe R.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '22814378',
          },
          {
            database: 'DOI',
            id: '10.1073/pnas.1210303109',
          },
        ],
        title:
          'N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.',
        publicationDate: '2012',
        journal: 'Proc. Natl. Acad. Sci. U.S.A.',
        firstPage: '12449',
        lastPage: '12454',
        volume: '109',
        completeAuthorList: true,
        literatureAbstract:
          'Protein N-terminal acetylation (Nt-acetylation) is an important mediator of protein function, stability, sorting, and localization. Although the responsible enzymes are thought to be fairly well characterized, the lack of identified in vivo substrates, the occurrence of Nt-acetylation substrates displaying yet uncharacterized N-terminal acetyltransferase (NAT) specificities, and emerging evidence of posttranslational Nt-acetylation, necessitate the use of genetic models and quantitative proteomics. NatB, which targets Met-Glu-, Met-Asp-, and Met-Asn-starting protein N termini, is presumed to Nt-acetylate 15% of all yeast and 18% of all human proteins. We here report on the evolutionary traits of NatB from yeast to human and demonstrate that ectopically expressed hNatB in a yNatB-Δ yeast strain partially complements the natB-Δ phenotypes and partially restores the yNatB Nt-acetylome. Overall, combining quantitative N-terminomics with yeast studies and knockdown of hNatB in human cell lines, led to the unambiguous identification of 180 human and 110 yeast NatB substrates. Interestingly, these substrates included Met-Gln- N-termini, which are thus now classified as in vivo NatB substrates. We also demonstrate the requirement of hNatB activity for maintaining the structure and function of actomyosin fibers and for proper cellular migration. In addition, expression of tropomyosin-1 restored the altered focal adhesions and cellular migration defects observed in hNatB-depleted HeLa cells, indicative for the conserved link between NatB, tropomyosin, and actin cable function from yeast to human.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '22814378',
          sourceCategories: ['PTM / Processing', 'Sequences'],
          referencePositions: [
            'ACETYLATION [LARGE SCALE ANALYSIS] AT MET-1 (ISOFORM 2)',
            'IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS]',
          ],
          referenceNumber: 19,
        },
      ],
      statistics: {
        reviewedProteinCount: 2292,
        unreviewedProteinCount: 3058,
        computationallyMappedProteinCount: 5,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '23186163',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Zhou H.',
          'Di Palma S.',
          'Preisinger C.',
          'Peng M.',
          'Polat A.N.',
          'Heck A.J.',
          'Mohammed S.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '23186163',
          },
          {
            database: 'DOI',
            id: '10.1021/pr300630k',
          },
        ],
        title:
          'Toward a comprehensive characterization of a human cancer cell phosphoproteome.',
        publicationDate: '2013',
        journal: 'J. Proteome Res.',
        firstPage: '260',
        lastPage: '271',
        volume: '12',
        completeAuthorList: true,
        literatureAbstract:
          'Mass spectrometry (MS)-based phosphoproteomics has achieved extraordinary success in qualitative and quantitative analysis of cellular protein phosphorylation. Considering that an estimated level of phosphorylation in a cell is placed at well above 100,000 sites, there is still much room for improvement. Here, we attempt to extend the depth of phosphoproteome coverage while maintaining realistic aspirations in terms of available material, robustness, and instrument running time. We developed three strategies, where each provided a different balance between these three key parameters. The first strategy simply used enrichment by Ti(4+)-IMAC followed by reversed chromatography LC-MS (termed 1D). The second strategy incorporated an additional fractionation step through the use of HILIC (2D). Finally, a third strategy was designed employing first an SCX fractionation, followed by Ti(4+)-IMAC enrichment and additional fractionation by HILIC (3D). A preliminary evaluation was performed on the HeLa cell line. Detecting 3700 phosphopeptides in about 2 h, the 1D strategy was found to be the most sensitive but limited in comprehensivity, mainly due to issues with complexity and dynamic range. Overall, the best balance was achieved using the 2D based strategy, identifying close to 17,000 phosphopeptides with less than 1 mg of material in about 48 h. Subsequently, we confirmed the findings with the K562 cell sample. When sufficient material was available, the 3D strategy increased phosphoproteome allowing over 22,000 unique phosphopeptides to be identified. Unfortunately, the 3D strategy required more time and over 1 mg of material before it started to outperform 2D. Ultimately, combining all strategies, we were able to identify over 16,000 and nearly 24,000 unique phosphorylation sites from the cancer cell lines HeLa and K562, respectively. In summary, we demonstrate the need to carry out extensive fractionation for deep mining of the phosphoproteome and provide a guide for appropriate strategies depending on sample amount and/or analysis time.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '23186163',
          sourceCategories: ['PTM / Processing', 'Sequences'],
          referenceComments: [
            {
              value: 'Erythroleukemia',
              type: 'TISSUE',
            },
          ],
          referencePositions: [
            'PHOSPHORYLATION [LARGE SCALE ANALYSIS] AT SER-2',
            'IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS]',
          ],
          referenceNumber: 20,
        },
      ],
      statistics: {
        reviewedProteinCount: 4864,
        unreviewedProteinCount: 8107,
        computationallyMappedProteinCount: 0,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '27995898',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Bonneau B.',
          'Ando H.',
          'Kawaai K.',
          'Hirose M.',
          'Takahashi-Iwanaga H.',
          'Mikoshiba K.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '27995898',
          },
          {
            database: 'DOI',
            id: '10.7554/elife.19896',
          },
        ],
        title:
          'IRBIT controls apoptosis by interacting with the Bcl-2 homolog, Bcl2l10, and by promoting ER-mitochondria contact.',
        publicationDate: '2016',
        journal: 'Elife',
        firstPage: 'e19896',
        lastPage: 'e19896',
        volume: '5',
        completeAuthorList: true,
        literatureAbstract:
          'IRBIT is a molecule that interacts with the inositol 1,4,5-trisphosphate (IP<sub>3</sub>)-binding pocket of the IP<sub>3</sub> receptor (IP<sub>3</sub>R), whereas the antiapoptotic protein, Bcl2l10, binds to another part of the IP<sub>3</sub>-binding domain. Here we show that Bcl2l10 and IRBIT interact and exert an additive inhibition of IP<sub>3</sub>R in the physiological state. Moreover, we found that these proteins associate in a complex in mitochondria-associated membranes (MAMs) and that their interplay is involved in apoptosis regulation. MAMs are a hotspot for Ca<sup>2+</sup> transfer between endoplasmic reticulum (ER) and mitochondria, and massive Ca<sup>2+</sup> release through IP<sub>3</sub>R in mitochondria induces cell death. We found that upon apoptotic stress, IRBIT is dephosphorylated, becoming an inhibitor of Bcl2l10. Moreover, IRBIT promotes ER mitochondria contact. Our results suggest that by inhibiting Bcl2l10 activity and promoting contact between ER and mitochondria, IRBIT facilitates massive Ca<sup>2+</sup> transfer to mitochondria and promotes apoptosis. This work then describes IRBIT as a new regulator of cell death.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '27995898',
          sourceCategories: [
            'Function',
            'PTM / Processing',
            'Disease & Variants',
            'Interaction',
            'Subcellular Location',
          ],
          referencePositions: [
            'FUNCTION',
            'INTERACTION WITH BCL2L10 AND ITPR1',
            'SUBCELLULAR LOCATION',
            'PHOSPHORYLATION AT SER-68; SER-71; SER-74 AND SER-77',
            'DEPHOSPHORYLATION',
            'MUTAGENESIS OF SER-68',
          ],
          referenceNumber: 21,
        },
        {
          source: {
            name: 'ORCID',
            id: '0000-0003-2302-5369',
          },
          citationId: '27995898',
          sourceCategories: ['Function', 'Interaction', 'Subcellular Location'],
          communityAnnotation: {
            proteinOrGene: 'IRBIT',
            function:
              'Regulates apoptosis through an interaction with the anti-apoptotic protein BCL2L10 and by acting on ER-mitochondria contact sites.',
            submissionDate: '2019-10-21',
          },
        },
      ],
      statistics: {
        reviewedProteinCount: 5,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 10,
        communityMappedProteinCount: 1,
      },
    },
    {
      citation: {
        id: '31799605',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Pinto R.',
          'Vaagboe C.B.',
          'Jakobsson M.E.',
          'Kim Y.',
          'Baltissen M.P.',
          "O'Donohue M.F.",
          'Guzman U.H.',
          'Malecki J.M.',
          'Wu J.',
          'Kirpekar F.',
          'Olsen J.V.',
          'Gleizes P.E.',
          'Vermeulen M.',
          'Leidel S.A.',
          'Slupphaug G.',
          'Falnes P.O.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '31799605',
          },
          {
            database: 'DOI',
            id: '10.1093/nar/gkz1147',
          },
        ],
        title:
          'The human methyltransferase ZCCHC4 catalyses N6-methyladenosine modification of 28S ribosomal RNA.',
        publicationDate: '2020',
        journal: 'Nucleic Acids Res.',
        firstPage: '830',
        lastPage: '846',
        volume: '48',
        completeAuthorList: true,
        literatureAbstract:
          'RNA methylations are essential both for RNA structure and function, and are introduced by a number of distinct methyltransferases (MTases). In recent years, N6-methyladenosine (m6A) modification of eukaryotic mRNA has been subject to intense studies, and it has been demonstrated that m6A is a reversible modification that regulates several aspects of mRNA function. However, m6A is also found in other RNAs, such as mammalian 18S and 28S ribosomal RNAs (rRNAs), but the responsible MTases have remained elusive. 28S rRNA carries a single m6A modification, found at position A4220 (alternatively referred to as A4190) within a stem-loop structure, and here we show that the MTase ZCCHC4 is the enzyme responsible for introducing this modification. Accordingly, we found that ZCCHC4 localises to nucleoli, the site of ribosome assembly, and that proteins involved in RNA metabolism are overrepresented in the ZCCHC4 interactome. Interestingly, the absence of m6A4220 perturbs codon-specific translation dynamics and shifts gene expression at the translational level. In summary, we establish ZCCHC4 as the enzyme responsible for m6A modification of human 28S rRNA, and demonstrate its functional significance in mRNA translation.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '31799605',
          sourceCategories: ['Interaction'],
          referencePositions: ['INTERACTION WITH ZCCHC4'],
          referenceNumber: 22,
        },
      ],
      statistics: {
        reviewedProteinCount: 8,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 1,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '28647132',
        citationType: 'UniProt indexed literatures',
        authors: [
          'Grbesa I.',
          'Kalo A.',
          'Beluzic R.',
          'Kovacevic L.',
          'Lepur A.',
          'Rokic F.',
          'Hochberg H.',
          'Kanter I.',
          'Simunovic V.',
          'Munoz-Torres P.M.',
          'Shav-Tal Y.',
          'Vugrek O.',
        ],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '28647132',
          },
          {
            database: 'DOI',
            id: '10.1016/j.ejcb.2017.05.002',
          },
        ],
        title:
          'Mutations in S-adenosylhomocysteine hydrolase (AHCY) affect its nucleocytoplasmic distribution and capability to interact with S- adenosylhomocysteine hydrolase-like 1 protein.',
        publicationDate: '2017',
        journal: 'Eur. J. Cell Biol.',
        firstPage: '579',
        lastPage: '590',
        volume: '96',
        completeAuthorList: true,
        literatureAbstract:
          'S-adenosylhomocysteine hydrolase (AHCY) is thought to be located at the sites of ongoing AdoMet-dependent methylation, presumably in the cell nucleus. Endogenous AHCY is located both in cytoplasm and the nucleus. Little is known regarding mechanisms that drive its subcellular distribution, and even less is known on how mutations causing AHCY deficiency affect its intracellular dynamics. Using fluorescence microscopy and GFP-tagged AHCY constructs we show significant differences in the intensity ratio between nuclei and cytoplasm for mutant proteins when compared with wild type AHCY. Interestingly, nuclear export of AHCY is not affected by leptomycin B. Systematic deletions showed that AHCY has two regions, located at both sides of the protein, that contribute to its nuclear localization, implying the interaction with various proteins. In order to evaluate protein interactions in vivo we engaged in bimolecular fluorescence complementation (BiFC) based studies. We investigated previously assumed interaction with AHCY-like-1 protein (AHCYL1), a paralog of AHCY. Indeed, significant interaction between both proteins exists. Additionally, silencing AHCYL1 leads to moderate inhibition of nuclear export of endogenous AHCY.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '28647132',
          sourceCategories: ['Interaction', 'Subcellular Location'],
          referencePositions: ['SUBCELLULAR LOCATION', 'INTERACTION WITH AHCY'],
          referenceNumber: 23,
        },
      ],
      statistics: {
        reviewedProteinCount: 2,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 8,
        communityMappedProteinCount: 0,
      },
    },
    {
      citation: {
        id: '25237103',
        citationType: 'UniProt indexed literatures',
        authors: ['Arnaoutov A.', 'Dasso M.'],
        citationCrossReferences: [
          {
            database: 'PubMed',
            id: '25237103',
          },
          {
            database: 'DOI',
            id: '10.1126/science.1251550',
          },
        ],
        title:
          'Enzyme regulation. IRBIT is a novel regulator of ribonucleotide reductase in higher eukaryotes.',
        publicationDate: '2014',
        journal: 'Science',
        firstPage: '1512',
        lastPage: '1515',
        volume: '345',
        completeAuthorList: true,
        literatureAbstract:
          'Ribonucleotide reductase (RNR) supplies the balanced pools of deoxynucleotide triphosphates (dNTPs) necessary for DNA replication and maintenance of genomic integrity. RNR is subject to allosteric regulatory mechanisms in all eukaryotes, as well as to control by small protein inhibitors Sml1p and Spd1p in budding and fission yeast, respectively. Here, we show that the metazoan protein IRBIT forms a deoxyadenosine triphosphate (dATP)-dependent complex with RNR, which stabilizes dATP in the activity site of RNR and thus inhibits the enzyme. Formation of the RNR-IRBIT complex is regulated through phosphorylation of IRBIT, and ablation of IRBIT expression in HeLa cells causes imbalanced dNTP pools and altered cell cycle progression. We demonstrate a mechanism for RNR regulation in higher eukaryotes that acts by enhancing allosteric RNR inhibition by dATP.',
      },
      references: [
        {
          source: {
            name: 'UniProtKB reviewed (Swiss-Prot)',
          },
          citationId: '25237103',
          sourceCategories: [
            'Function',
            'Family & Domains',
            'PTM / Processing',
            'Disease & Variants',
            'Interaction',
            'Structure',
          ],
          referencePositions: [
            'X-RAY CRYSTALLOGRAPHY (2.64 ANGSTROMS) OF 89-530 OF MUTANT ALA-508 IN COMPLEX WITH NAD',
            'COFACTOR',
            'FUNCTION',
            'INTERACTION WITH RRM1',
            'MISCELLANEOUS',
            'PEST REGION',
            'PHOSPHORYLATION',
          ],
          referenceNumber: 24,
        },
      ],
      statistics: {
        reviewedProteinCount: 2,
        unreviewedProteinCount: 0,
        computationallyMappedProteinCount: 6,
        communityMappedProteinCount: 0,
      },
    },
  ],
};

export default mock;
